src/Assets/Scripts/Player/PlayerController.cs

```diff
@@ -7,6 +7,9 @@
 [RequireComponent(typeof(Rigidbody2D))]
 public class PlayerController : MonoBehaviour
 {
+    [SerializeField]
+    public PolygonCollider2D[] colliders;
+    public int currentColliderIndex = 0;
```

Comment:
Denna ska definitivt inte vara public.
```suggestion
    private int currentColliderIndex = 0;
```